api/ANGLE_instanced_arrays.json

```diff
@@ -205,4 +193,4 @@
       }
     }
   }
-}
+}
```

Comment:
@Elchi3 One preliminary question: is the removal of the trailing CRs an intended feature of the PR?

Reply:
Good catch! It was caused by the script I used. But you are right, as stated in the [.editorconfig](https://github.com/mdn/browser-compat-data/blob/master/.editorconfig) we should have a newline (LF). I fixed this now.